pkg/account/api/validation.go

```diff
@@ -37,17 +37,21 @@ func verifyEmailFormat(email string) bool {
 }
 
 func validateCreateAPIKeyRequest(req *accountproto.CreateAPIKeyRequest, localizer locale.Localizer) error {
-	if req.Command == nil {
```

Comment:
The old console is still using command, we can keep this check for now, and add `req.Command.Name == ""` below this.

After we fully migrate to this, we can remove the check.

Reply:
I understand, but because I add a logic to check if command == nil then it goes to s.createAPIKeyNoCommand https://github.com/bucketeer-io/bucketeer/pull/1344/files#diff-1236610833a324b8af6d6273437bdeb2fb35012b5ae3b3374760927a8427c204R53.
So if it is not null it will go to that old validation function, and the condition is always false!

Reply:
Nice logic!